app/views/users/show.html.erb

```diff
@@ -1 +1,29 @@
-<h1>Welcome to the show of users</h1>
+<div class="container py-3">
+  <%=link_to '< Back to Users', users_url, class: 'title-link' %>
+
+  <div class="card shadow user-card">
+    <div class="card-body p-2">
+      <img src="<%=@user.photo || image_url('empty_user.png') %>" alt="<%=@user.name %>" title="<%=@user.name %>" class="user-image user-image-profile" />
+      <h3 class="card-title"><%=@user.name %></h3>
+      <hr class="mt-4" />
+      <h5 class="card-title">Bio</h5>
+      <p class="card-text"><%=@user.bio %></p> 
+    </div>
+  </div>
+
+  <h5 class="my-3">Posts (<%=@user.posts_counter %>)</h5>
+
+  <% @user.posts.order(id: :desc).first(5).each do |post| %>
```

Comment:
- [ ] As per this [requirement](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/projects/Views_project.md#:~:text=Use%20methods%20that%20you%20have%20created%20in%20the%20Blog%20App%20%2D%20processing%20data%20in%20models.), Please use `last_three_posts` method you created in user model.

Reply:
you are absolutely right, I completely forgot those methods, I just applied them to the code, thanks